requirements.txt

```diff
@@ -12,4 +12,4 @@ numpy
 nmeatoolkit
 matplotlib
 weatherrouting
-gdal==3.6.4
\ No newline at end of file
+GDAL
```

Comment:
this breaks CI; why did you removed the version pin?

Reply:
Ah, it's because I tested a newer version and everything seemed to work. Do you want to keep the specific version for a purpose or I can use the version pin for the latest version?

Because all the other packages except for eecodes do not have version pins.

Reply:
i used GDAL==3.9.1, which is a version that is still compatible.
3.9.2 fails indeed to install, which is quite weird.

Reply:
3f9ac625206260c8b43a41e9dd41d0d94cff79a9

Reply:
Ok great 👍 we need to keep the CI working in order to assurde that it will work at least on an ubuntu/debian system 👍

Reply:
CI still fails also with 3.9.1 https://github.com/dakk/gweatherrouting/actions/runs/10434534328/job/28905597460?pr=135